package gamer.protocol;

message MyLoginMsgProtocol {
  required string account  = 1;
  required int32 password  = 2;
  optional int32 code      = 3[default = 0];
}